TripHouse/main.js

```diff
@@ -51,17 +51,60 @@ const data = [
     },
   ];
 
-const homesContent = document.querySelector('.homes_content')
-
+const homesContent = document.querySelector('.homes_content');
 function createCard1(data) {
   return `
   <div class="content_img col-xs-6 col-sm-6">
     <img src="${data.imageUrl}" alt="${data.name}">
     <p class="accent-blue">${data.name}</p>
     <p class="secondary">${data.city}, ${data.country}</p>
   </div>`
+};
+data.forEach((element, index) => {
+  if (index < 4) {
+    homesContent.insertAdjacentHTML('beforeend', createCard1(element))
+  }
+});
+
+const homesImgArrowRight = (homesContent.getElementsByClassName('content_img')[3]);
+function createArrowRigt() {
+  return `<img class="arrow_right" src="./img/arrow-right.png" alt="arrow-right">`
+};
+homesImgArrowRight.insertAdjacentHTML('beforeend', createArrowRigt())
+
+const homesImgArrowLeft = (homesContent.getElementsByClassName('content_img')[0])
+function createArrowLeft() {
+  return `<img class="arrow_left" src="./img/arrow-left.png" alt="arrow-leftt">`
+};
```

Comment:
Why you add it with js when arrows are static?

Reply:
As I wrote earlier, I mixed up branches and 10 and 11 tasks are finished in branch 11, comments for branch 10 are made in branch 11